packages/flutter/lib/src/widgets/route_notification_messages.dart

```diff
@@ -1,41 +0,0 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I just realized we are not exposing this api in the package level, we should be good to remove this without a breaking change